src/app/shapes/Triangle.ts

```diff
@@ -1,90 +1,78 @@
-var stage = new Konva.Stage({
-      container: 'container',
+import * as Konva from "konva"; 
+
+let stage = new Konva.Stage({
+      container: "container",
       width: window.innerWidth,
       height: window.innerHeight
-    });
+});
     
-    const layer = new Konva.Layer();
+const layer = new Konva.Layer();
 stage.add(layer);
 
 class Triangle {
-
-	public instance: Konva.Shape;
-  public left: number;
-  public top: number;
+    public instance: Konva.Shape;
+    public left: number = 0;
+    public top: number = 0;
   
-  public constructor(x1:number, y1:number, x2:number, y2:number, x3:number, y3:number) {
-    if(x1 < x2 && x1 < x3) {
-    		this.left = x1;
-    }
-    if(x2<x1 && x2<x3) {
-    	  this.left = x2;
-    }
-    if(x3<x1 && x3<x1){
-    		this.left = x3;
-    }
-    if(y1 < y2 && y1 < y3) {
-    	 this.top = y1;
+    public constructor(x1:number, y1:number, x2:number, y2:number, x3:number, y3:number) {
+        if(x1 < x2 && x1 < x3) {
+            this.left = x1;
+        }
+        if(x2<x1 && x2 < x3) {
+            this.left = x2;
+        }
+        if(x3<x1 && x3 < x1){
+            this.left = x3;
+        }
+        if(y1 < y2 && y1 < y3) {
+            this.top = y1;
+        }
+        if(y2 < y1 && y2 < y3) {
+            this.top = y2;
```

Comment:
This can be simplified to `this.left = Math.min(x1, x2, x3); ` and `this.top = Math.min(y1, y2, y3);`